src/Compilers/Test/Core/Diagnostics/DiagnosticDescription.cs

```diff
@@ -36,6 +37,7 @@ public sealed class DiagnosticDescription
         private readonly bool _ignoreArgumentsWhenComparing;
         private readonly DiagnosticSeverity? _defaultSeverityOpt;
         private readonly DiagnosticSeverity? _effectiveSeverityOpt;
+        private readonly ImmutableArray<string>? _originalFormatSpecifiers;
```

Comment:
>ImmutableArray<string>? [](http://example.com/codeflow?start=25&length=23)

We typically avoid `ImmutableArray<T>?` and use `default(ImmutableArray<T>)` for an optional value instead.